dataplane/linux/int_dataplane.go

```diff
@@ -258,7 +260,7 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 		applyThrottle:     throttle.New(10),
 	}
 	dp.applyThrottle.Refill() // Allow the first apply() immediately.
-
+	dp.hostname = config.Hostname
```

Comment:
Do we need this change? I can't seem to see where we use this, but could be missing it.

Reply:
No we don't, I'll remove it